### DISCUSSION TITLE
Getting started with UI Kit

### DISCUSSION THREAD

**Discussion by Leonardonline** [primefaces/community#3119] [Figma UI Kit]
Hello, everyone!

I recently purchased the PrimeFaces UI Kit and am eager to start integrating it into my workflow. However, I have a few questions and would love some insights from the community:

Basic Setup: What are the initial steps to get started with this UI Kit? Are there any specific recommendations or best practices for setting it up efficiently in design tools?

Does this UI Kit require a Pro version of Figma, or is it fully functional with the free version?

Is Token Studio necessary to use this UI Kit effectively? If so, are there specific configurations or features I should focus on?

Any guidance, resources, or tips would be immensely helpful!

Thank you in advance!

**Comment by onursenture** [comment]
Hi, first of all, thank you for choosing the UI Kit. We've tried to address your questions on the Get Started page included in the kit:

<img width="2074" alt="image" src="https://github.com/user-attachments/assets/79b2c7f0-f8ff-49d4-8c89-fa95e6c7ef91">

We recommend checking out the Getting Started section on this page, along with the Figma Libraries, Using PrimeOne, Adding PrimeIcons, and Tokens Studio sections, which are explained with screenshots.

If I remember correctly, Figma's free tier still allows you to use the UI Kit, but you won't be able to publish libraries. Therefore, if you plan to use components from the UI Kit in other files, you might need a paid Figma tier.
